contrib/cirrus/runner.sh

```diff
@@ -21,16 +21,13 @@ source $(dirname $0)/lib.sh
 
 showrun echo "starting"
 
+function _run_lint() {
+    make lint
+}
+
 function _run_validate() {
-    # git-validation tool fails if $EPOCH_TEST_COMMIT is empty
-    # shellcheck disable=SC2154
-    if [[ -n "$EPOCH_TEST_COMMIT" ]]; then
-        showrun make validate
-    else
-        warn "Skipping git-validation since \$EPOCH_TEST_COMMIT is empty"
-    fi
-    # make sure PRs have tests
-    showrun make tests-included
+    # These targets require/assume binaries and documentation has been built
+    showrun make .gitvalidation validate.completions man-page-check swagger-check tests-expect-exit pr-removes-fixed-skips
```

Comment:
Ooof. Not only is this unmaintainable duplication, it's also a POLA violation: a CI job called `validate`, which calls a function named `_run_validate()`, should run `make validate`. I doubt we can just remove `lint` from `make validate`, because we don't know who depends on that, so the only option I can think of is to make a new `make validate-nolint` target and refactor as needed (i.e., `validate: lint validate-nolint`).

Also, `make tests-included` got lost.

Reply:
agreed it would be good to have make validate split. But I would say we need validate-build and validate-sources see my other comment

Reply:
I'm open to renaming the tasks as well, that's pretty easy.

Ack. re: new make targets, that's a better way to go.